RInAppMessaging/Classes/Utilities/AtomicWrapper.swift

```diff
@@ -0,0 +1,29 @@
+/// This wrapper ensures synchronized access to the value only for getter and setter.
+/// Mutating functions, subscript, incrementation etc. are not synchronized.
+@propertyWrapper
+struct AtomicGetSet<Value> {
+    private let queue = PropertyQueueGenerator.spawnQueue(domain: "IAM.AtomicProperty")
```

Comment:
Does each struct needs its own queue?

Reply:
To ensure independent access between different properties

Reply:
To ensure independent access between different properties